### DISCUSSION TITLE
`initiate()`: is `isError` always `false` if `isSuccess` is `true` and vice versa?

### DISCUSSION THREAD

**Discussion by pseidemann** [reduxjs/redux-toolkit#4623] [Q&A] [answered]
I'm wondering if this statement always holds true (in both directions) when using `await dispatch(someEndpoint.initiate())`:

$$
\mathrm{isError} = \neg\mathrm{isSuccess}
$$

maybe it's possible that both are false? I couldn't find anything about that in the docs

**Comment by phryneas** [comment, marked as answer]
Those two are exclusive from each other. You can have an error and data, though.
